webpack/scenes/ContentViews/Publish/__tests__/publishContentView.test.js

```diff
@@ -0,0 +1,127 @@
+import React from 'react';
+import * as reactRedux from 'react-redux';
+import { renderWithRedux, patientlyWaitFor, fireEvent } from 'react-testing-lib-wrapper';
+
+import { nockInstance, assertNockRequest } from '../../../../test-utils/nockWrapper';
+import api from '../../../../services/api';
+import PublishContentViewWizard from '../PublishContentViewWizard';
+
+const cvDetailData = require('../../Details/__tests__/contentViewDetails.fixtures.json');
+
+const cvPublishPath = api.getApiUrl('/content_views/1/publish');
+const publishResponseData = require('./publishResponse.fixture.json');
+
+const environmentPathsPath = api.getApiUrl('/organizations/1/environments/paths');
+const environmentPathsData = require('./environmentPaths.fixtures.json');
+
+test('Can call API and show Wizard', async (done) => {
```

Comment:
Would like to see tests added

* `Shows a warning when environments are selected out of order`
* `Doesn't show a warning when environments selected are the next in the lifecycle path`

Reply:
Added a test for in order and out of order env. 👍🏽